invokeai/app/services/urls/urls_default.py

```diff
@@ -15,3 +15,6 @@ def get_image_url(self, image_name: str, thumbnail: bool = False) -> str:
             return f"{self._base_url}/images/i/{image_basename}/thumbnail"
 
         return f"{self._base_url}/images/i/{image_basename}/full"
+
+    def get_model_image_url(self, model_key: str) -> str:
+        return f"{self._base_url}/model_images/{model_key}.png"
```

Comment:
Wouldn't this be `f"{self._base_url}/models/i/{model_key}/image"`? Maybe I'm missing something